lib/Onyx.js

```diff
@@ -154,7 +151,7 @@ function removeLastAccessedKey(key) {
  */
 function addLastAccessedKey(key) {
     // Only specific keys belong in this list since we cannot remove an entire collection.
-    if (isCollectionKey(key) || !isSafeEvictionKey(key)) {
+    if (isCollectionKey(key)) {
```

Comment:
This was updated to allow to keep more items in the access list
To preserve the same functionality the check was moved here: [Line 481](https://github.com/Expensify/react-native-onyx/pull/88/files#diff-146c57cda221f8f5ee5c808eee0c6692b02c4ac2b8818d0104896a04be78e78aR481)